### ISSUE TITLE
Shorten the name of cmdlets

### ISSUE THREAD

**Issue by rezanid** [rezanid/PSDataverse#24]
Currently all Cmdlets use the following pattern for naming:
```
<verb>-Dataverse<noun>
```
For operations that have long noun part in the name like `Get-DataverseRowCount`, it can be tedious to type such a long name.

Consider renaming all the existing Cmdlets to the following pattern:
```
<verb>-Dv<noun>
```

To continue supporting the current names, old functions can become wrappers to call new functions.
```
Function Clear-DvTable {
    # Implementation here
}
Function Clear-DataverseTable {
  Clear-DvTable
}
```

**Comment by rvdwegen** [comment]
Generally in Powershell names of functions and operations are verbose, with a shorter alias as alternative. Would that not be a better solution?

**Comment by rezanid** [comment]
> Generally in Powershell names of functions and operations are verbose, with a shorter alias as alternative. Would that not be a better solution?

Indeed, that's a good proposition. Though we will need to good aliases that are short yet distinctive enough.
